service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/cmo/DraftOrdersApprovedEventHandler.java

```diff
@@ -143,32 +185,55 @@ public void sendNotificationToCafcassViaSendGrid(final DraftOrdersApproved event
         CaseData caseData = event.getCaseData();
 
         if (CafcassHelper.isNotifyingCafcassEngland(caseData, cafcassLookupConfiguration)) {
+            List<HearingOrder> approvedOrders = new ArrayList<>();
+            approvedOrders.addAll(event.getApprovedOrders());
+            approvedOrders.addAll(getConfidentialOrderToBeSentToCafcass(event));
+
+            if (approvedOrders.isEmpty()) {
+                log.info("Only confidential orders uploaded and not accessible by Cafcass. "
```

Comment:
Looks like missing test case for this scenario.

Reply:
covered already. `shouldNotSendToCafcassEnglandOfApprovedConfidentialOrderIfNotUploadedByChildSolicitor`